c++/src/capnp/compat/json-test.c++

```diff
@@ -214,7 +214,7 @@ KJ_TEST("decode all types") {
   CASE(R"({"int32Field":"2147483647"})", root.getInt32Field() == 2147483647);
   CASE(R"({"int64Field":-9007199254740992})", root.getInt64Field() == -9007199254740992LL);
   CASE(R"({"int64Field":9007199254740991})", root.getInt64Field() == 9007199254740991LL);
-  CASE(R"({"int64Field":"-9223372036854775808"})", root.getInt64Field() == -9223372036854775808LL);
+  CASE(R"({"int64Field":"-9223372036854775808"})", root.getInt64Field() == -9223372036854775808ULL);
```

Comment:
I have no idea why both gcc and clang warn when this constant is LL. Changing to ULL fixed integer range warning.